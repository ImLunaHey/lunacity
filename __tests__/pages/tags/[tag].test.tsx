import { render } from '@testing-library/react';
import '@testing-library/jest-dom';
import Tag from '../../../src/pages/tags/[tag]';

describe('Tag', () => {
    it('renders', () => {
        const { container } = render(<Tag tag='test-tag' />);
        expect(container).toMatchSnapshot()
    });
});
