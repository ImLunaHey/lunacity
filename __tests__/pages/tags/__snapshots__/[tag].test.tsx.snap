// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Tag renders 1`] = `
<div>
  <div>
    test-tag
  </div>
</div>
`;
